waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/model/MetaStoreMappingImpl.java

```diff
@@ -78,11 +84,18 @@ public String getDatabasePrefix() {
 
   @Override
   public void close() throws IOException {
-    client.close();
```

Comment:
Does this still need to be closed explicitly at some point?

Reply:
Connection will be monitored so it will be closed, but probably best to do it in the pool and poolFactory explicitly as well. Good practice